src/applications/personalization/profile360/manifest.json

```diff
@@ -3,4 +3,4 @@
   "entryFile": "./profile-360-entry.jsx",
   "entryName": "profile-360",
   "rootUrl": "/profile"
-}
+}
```

Comment:
```suggestion

```
No need to remove the end of line.

Reply:
Hmm, I believe it's a setting in my linter - it automatically removes empty lines at the end of the file. Let me see if I can change the setting.